install_sct

```diff
@@ -175,6 +175,26 @@ function fetch_os_type() {
   fi
 }
 
+# Source: https://stackoverflow.com/a/68148776
+function fetch_arch() {
+  uname_arch="$(uname -m)"  # -i is only linux, -m is linux and apple
+  if [[ "$uname_arch" = x86_64* ]]; then
+    if [[ "$(uname -a)" = *ARM64* ]]; then
+      ARCH='a64'
+    else
+      ARCH='x64'
+    fi
+  elif [[ "$uname_arch" = i*86 ]]; then
+    ARCH='x32'
+  elif [[ "$uname_arch" = arm* ]]; then
+    ARCH='a32'
+  elif test "$uname_arch" = aarch64; then
+    ARCH='a64'
+  else
+    exit 1
+  fi
+}
+
```

Comment:
Oops, please ignore this. I committed this accidentally. :sweat: 

Removing in a follow-up commit...